src/graphql/resolvers/index.js

```diff
@@ -7,6 +7,7 @@ import mutations, { fieldResolvers as mFieldResolvers } from './mutations';
 const createServer = {
   ...graphScalars,
   ...graph.scalars.date,
+  ...graph.scalars.slug,
```

Comment:
I think here we can just `...graph.scalars` and it will spread in date and slug

Reply:
@csell5 That didn't work for me when working with the members project. I had to put it this way.

Reply:
👍